pkg/downloader/chart_downloader.go

```diff
@@ -139,7 +156,7 @@ func (c *ChartDownloader) DownloadTo(ref, version, dest string) (string, *proven
 //		* If version is non-empty, this will return the URL for that version
 //		* If version is empty, this will return the URL for the latest version
 //		* If no version can be found, an error is returned
-func (c *ChartDownloader) ResolveChartVersion(ref, version string) (*url.URL, getter.Getter, error) {
+func (c *ChartDownloader) ResolveChartVersion(ref, version string) (*url.URL, *repo.ChartRepository, error) {
```

Comment:
Looks like this will be a backwards incompatible change because we're changing the return values.

Reply:
Fixed: 
The old signature of the *ResolveChartVersion* is back.
I added a new method - *ResolveChartVersionAndGetRepo*, with the new returned values.